### DISCUSSION TITLE
I need some help with building

### DISCUSSION THREAD

**Discussion by no-usernamefr** [Eloston/ungoogled-chromium#2877] [Q&A]
I already have the source of chromium and depot-tools installed. But the build instructions are not very clear on what folder to run the command in, do I run it in the depot-tools folder, or chromium?

**Comment by Qubik65536** [comment]
On which platform you're trying to build on? Anyway, you have to follow the platform-specific instructions like the one for [Windows](https://github.com/ungoogled-software/ungoogled-chromium-windows) or [macOS](https://github.com/ungoogled-software/ungoogled-chromium-macos). Ungoogled-Chromium doesn't use any Google provided build tools (such as depot-tools).

**Reply by no-usernamefr** [reply]
I'm using windows. Since I have the chromium source installed, I could delete the depot-tools? And from my understanding I would just open the folder with the chromium source into Visual Studio, and run the build commands on the page you linked?

**Reply by Qubik65536** [reply]
Just follow the instruction. It will tell you how to configure everything from your Visual Studio to the code repository.

**Reply by no-usernamefr** [reply]
Thanks, do you think it would probably be best for me to start over. And delete the chromium source and depot tools?

**Reply by Qubik65536** [reply]
Yes. To build with those instructions, as far as I can tell, you don't need to download Chromium source and `depot_tools` (which is not **even** used) by yourself, actually the build script that will be used (if you follow the instruction) will download the source code by itself, not use the one that you downloads.
